tests/test_docs.py

```diff
@@ -254,7 +254,11 @@ def run_block_test(self, block):
                 if lang == 'edgeql':
                     ql_parser.parse_block(snippet)
                 elif lang == 'sdl':
-                    ql_parser.parse_sdl(f'module default {{ {snippet} }}')
+                    # the snippet itself may or may not contain a module block
+                    if 'module' in snippet:
```

Comment:
This is rather fragile check. Add a simple regexp to check for `module\s+\w+\s+{`

Reply:
Fair enough.